compose/service.py

```diff
@@ -479,6 +488,7 @@ def build(self, no_cache=False):
             stream=True,
             rm=True,
             nocache=no_cache,
+            fileobj=self._get_fileobj(),
```

Comment:
Within `docker-py` there's a `utils.compare_version`
``` python
def utils.def compare_version(v1, v2):
    """Compare docker versions

    >>> v1 = '1.9'
    >>> v2 = '1.10'
    >>> compare_version(v1, v2)
    1
    >>> compare_version(v2, v1)
    -1
    >>> compare_version(v2, v2)
    0
    """
    s1 = StrictVersion(v1)
    s2 = StrictVersion(v2)
    if s1 == s2:
        return 0
    elif s1 > s2:
        return -1
    else:
        return 1
```

Do you know if there's a helper like this already within compose so I could add something like :
```python
        if 'dockerfile' in options and utils.compare_version('1.17', client.api_version()) is -1:
            raise ConfigError('Docker client API must be greater than 1.17 to support the dockerfile option')
```

and when building:
``` python
    def build(self, no_cache=False):
        log.info('Building %s...' % self.name)

        build_kwars = dict(
            tag=self.full_name,
            stream=True,
            rm=True,
            nocache=no_cache,
        )
        if 'dockerfile' in self.options and \
                utils.compare_version('1.17', self.client.api_version()) is not -1:
            build_kwars['dockerfile'] = self.options.get('dockerfile', None)

        build_output = self.client.build(
            self.options['build'],
            **build_kwars
        )
```